engine/gamesys/src/gamesys/scripts/script_physics.cpp

```diff
@@ -1073,6 +1119,9 @@ namespace dmGameSystem
         {"set_gravity",     Physics_SetGravity},
         {"get_gravity",     Physics_GetGravity},
 
+        {"get_velocity_iteration",     Physics_GetVelocityIteration},
+        {"get_position_iteration",     Physics_GetPositionIteration},
```

Comment:
What is the use case of getting these values in Lua?

Reply:
Just to know that I can get out the value correctly after modified in game.project